lib/routes/account.js

```diff
@@ -409,7 +409,7 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
         const email = form.email
         const authPW = form.authPW
         const originalLoginEmail = request.payload.originalLoginEmail
-        const verificationMethod = request.payload.verificationMethod || request.query.verificationMethod
+        let verificationMethod = request.payload.verificationMethod || request.query.verificationMethod
```

Comment:
@vbudhram what's the use case for providing verificationMethod via query param?

Reply:
Looked at the javascript client and it passes this in through body params. Will remove check for query.